src/providers/execution/contracts/staking_router.py

```diff
@@ -58,3 +58,30 @@ def get_all_node_operator_digests(self, module: StakingModule, block_identifier:
 
 class StakingRouterContractV2(StakingRouterContract):
     abi_path = './assets/StakingRouterV2.json'
+
+    @lru_cache(maxsize=1)
+    @list_of_dataclasses(StakingModule.from_response)
+    def get_staking_modules(self, block_identifier: BlockIdentifier = 'latest') -> list[StakingModule]:
+        """
+        Returns all registered staking modules
+        """
+        contract_version = self.get_contract_version(block_identifier)
+
+        if contract_version == 1:
+            # Backward compatibility in case if new oracle have to build report for old protocol version
+            # But latest contracts has new version
+            staking_router = self.w3.eth.contract(
```

Comment:
let's log that we call an old abi. here or instead of logging at the end of function

Reply:
fixed